samples/crypto/eddsa/README.rst

```diff
@@ -0,0 +1,55 @@
+.. _crypto_eddsa:
+
+Crypto: EdDSA
+#############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The EdDSA sample shows how to sign and verify messages using the Edwards25519 curve.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+.. include:: /includes/tfm.txt
+
+Overview
+********
+
+The sample performs the following operations:
+
+1. Initialization:
+
+   a. The Platform Security Architecture (PSA) API is initialized.
+   #. A random Elliptic Curve Cryptography (ECC) key pair is generated and imported into the PSA crypto keystore.
```

Comment:
This is misleading. It is only the private key of the key pair that is imported into the keystore at this point no?

The way this (and the below) is written makes it sounds like there are 3 keys in the store. But it's just one pair right?

Reply:
I thought the public key is derived/calculated from the private key?

How would you describe a situation where both a private key and it's public key is imported into the PSA crypto keystore?

Reply:
> I thought the public key is derived/calculated from the private key?
> 
Well on key generation you create them as a pair, but given a private key you can derive the public key.


> How would you describe a situation where both a private key and it's public key is imported into the PSA crypto keystore?

That means importing a  key pair.

Reply:
That looks much better.

It is more clear that at the end we will have a private key and it's corresponding public key physically stored in the key store.

Reply:
So, the sample is actually trying to demonstrate being both sides of the usage-case in one single main file.
Having two entries in the keystore is not needed to sign and verify the message.
The sample is showing how to generate the key, and then give it to the other "side".
And how the other "side" takes this key and then use it.